packages/lean-imt/tests/lean-imt.test.ts

```diff
@@ -335,54 +335,43 @@ describe("Lean IMT", () => {
         it("Should export a tree", () => {
             const tree = new LeanIMT(poseidon, leaves)
 
-            const exportedTree = tree.export()
+            const nodes = tree.export()
```

Comment:
Could use some tests of trees with types other than bigint, and with different reviver funtions.